omod/src/main/java/org/openmrs/module/patientflags/extension/html/FlagPatientDashboardHeaderExt.java

```diff
@@ -43,14 +43,9 @@ public String getOverrideContent(String test) {
 		
 		if (!results.isEmpty()) {
 			String content = "<TABLE><TR><TD>";
-			
-			// comment out the header text for now
-			//        + Context.getMessageSourceService().getMessage("patientflags.flagpatientdashboard.header") + " ";
-			
 			for (Flag flag : results) {
-				content = content + "<SPAN " + (flag.getPriority() != null ? flag.getPriority().getStyle() : "") + ">" + flag.getLocalizedMessage() + "</SPAN>, ";
+				content = content + "<SPAN " + (flag.getPriority() != null ? flag.getPriority().getStyle() : "") + ">" + flag.getName() + "</SPAN>, ";

```

Comment:
Ah, okay, that makes sense with a couple caveats:

1) I'd change this so that it supports a localized name

2) When the next version is released (probably version 1.4?) in the release notes this change should be noticed, so that any existing users are aware--it's possible that they are using non-display-friendly flag names.